src/adapters/mc/OsConfigPolicy_DeployIfNotExists.json

```diff
@@ -401,25 +641,89 @@
                                                 "name": "OsConfigPolicy",
                                                 "version": "1.0.0",
                                                 "contentType": "Custom",
-                                                "contentUri": "<SAS token URL for the artifacts package>",
-                                                "contentHash": "<file hash for the artifacts package>",
+                                                "contentUri": "https://gcpoc88.blob.core.windows.net/blobforosconfig/LinuxSshServerSecurityBaselinePolicy.zip?sp=r&st=2024-02-20T19:49:37Z&se=2034-02-21T03:49:37Z&sv=2022-11-02&sr=b&sig=fiZZf8ML276R7PLkpGvYgwHZKIB9e67WU%2BcFAhs%2BACE%3D",
```

Comment:
Same here. Is this exposed intentionally?

Reply:
Good point. It has to be replaced as before. It's a link to the package and its hash, no big deal, but not proper to sit in the main either. Will fix.